config/routes.rb

```diff
@@ -50,6 +50,7 @@
       end
     end
     resources :submissions,       except: %i[new],  controller: 'grant_submissions/submissions' do
+      resources 'submission_applicants', only: %i[index create destroy], controller: 'grant_submissions/submissions/submission_applicants'
```

Comment:
shouldn't resources be `applicants`? in that case path would be `/submissions/1/applicants` rather than `/submissions/1/submission_applicants`